gym/envs/registration.py

```diff
@@ -96,6 +96,7 @@ class EnvSpec:
     max_episode_steps: Optional[int] = field(default=None)
     order_enforce: bool = field(default=True)
     autoreset: bool = field(default=False)
+    return_two_dones: bool = field(default=False)
```

Comment:
I don't think this will actually be useful. It only really has any value if people are explicitly setting it, which is a new (and very temporary) feature, and we explicitly deprecate one of the settings the moment it's introduced. This goes back to the general comment that I think the library should only support one step semantic, and compatibilities can be done through a wrapper-like thing

Reply:
Do you mean a wrapper that's always applied at make is preferable to an argument? Or no wrapper at make? My point was to have this to allow backward compatibility and, if users so chose to use the new api, they could just switch it to True at make.

Reply:
I think I just find it confusing as to what this is supposed to do. Does this indicate what the environment intrinsically returns? Does it decide whether a wrapper should be applied?

I'd prefer a wrapper that's always applied, unless explicitly turned off, which defaults to a no-op if the environment already follows the new API